lbrynet/lbrynet_daemon/LBRYDaemon.py

```diff
@@ -40,6 +40,7 @@
 from lbrynet.lbrynet_daemon.LBRYPublisher import Publisher
 from lbrynet.lbrynet_daemon.LBRYExchangeRateManager import ExchangeRateManager
 from lbrynet.lbrynet_daemon.Lighthouse import LighthouseClient
+from lbrynet.lbrynet_daemon.auth.server import LBRYJSONRPCServer, auth_required, authorizer
```

Comment:
lets not use LBRY in names. all this stuff is LBRY stuff. its part of the lbrynet module. there's no reason to repeat that all over the place.

if you're in a refactoring mood, it would be great to remove it basically everywhere (e.g. LBRYPublisher, LBRYExchangeRateManager, etc)